src/main/java/com/terraformersmc/modmenu/gui/widget/ModListWidget.java

```diff
@@ -219,20 +214,33 @@ protected void renderList(DrawContext DrawContext, int mouseX, int mouseY, float
 					RenderSystem.setShader(GameRenderer::getPositionProgram);
 					RenderSystem.setShaderColor(float_2, float_2, float_2, 1.0F);
 					Matrix4f matrix = DrawContext.getMatrices().peek().getPositionMatrix();
-					buffer.begin(VertexFormat.DrawMode.QUADS, VertexFormats.POSITION);
-					buffer.vertex(matrix, entryLeft, entryTop + entryHeight + 2, 0.0F).next();
-					buffer.vertex(matrix, selectionRight, entryTop + entryHeight + 2, 0.0F).next();
-					buffer.vertex(matrix, selectionRight, entryTop - 2, 0.0F).next();
-					buffer.vertex(matrix, entryLeft, entryTop - 2, 0.0F).next();
-					tessellator.draw();
+					BuiltBuffer builtBuffer;
+					buffer = tessellator.begin(VertexFormat.DrawMode.QUADS, VertexFormats.POSITION);
+					buffer.vertex(matrix, entryLeft, entryTop + entryHeight + 2, 0.0F);
+					buffer.vertex(matrix, selectionRight, entryTop + entryHeight + 2, 0.0F);
+					buffer.vertex(matrix, selectionRight, entryTop - 2, 0.0F);
+					buffer.vertex(matrix, entryLeft, entryTop - 2, 0.0F);
+					try
+					{
+						builtBuffer = buffer.end();
+						BufferRenderer.drawWithGlobalProgram(builtBuffer);
+						builtBuffer.close();
+					}
+					catch (Exception ignored) {}
```

Comment:
Nitpick: braces should be on the same lines as keywords.

```suggestion
					try {
						builtBuffer = buffer.end();
						BufferRenderer.drawWithGlobalProgram(builtBuffer);
						builtBuffer.close();
					} catch (Exception e) {
						// Ignored
					}
```